apps/general/serializers.py

```diff
@@ -1 +1,52 @@
-#Create Serializers
\ No newline at end of file
+from rest_framework import serializers
+from .models import *
+from apps.masters.serializers import Mod_StatusesSerializer
+import os
+
+class Mod_CompaniesSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Companies
+        fields = ['company_id','name']
+
+class Mod_BranchesSerializer(serializers.ModelSerializer):
```

Comment:
Did you create this class name? why it is in snakecase instead of camlecase

Reply:
yes Thulasi actually I have taken reference from HRMS I will chnage it to camlecase Thulasi.